File: hw/gudino/u3/hw28PrinciplesAplication/ComputerStore/productos_app/models/product.py
from dataclasses import dataclass

@dataclass
class Product:
    name: str
    make: str
    base_price: float
    final_price: float
    
    def to_dict(self):
        return {
            "name": self.name,
	    "make": self.make,
            "base_price": self.base_price,
            "final_price": self.final_price
        }

File: hw/gudino/u3/hw28PrinciplesAplication/ComputerStore/compujs/models/Product.js
class Product {
    constructor(name, make, basePrice, finalPrice) {
        this.name = name;
        this.make = make;
        this.basePrice = basePrice;
        this.finalPrice = finalPrice;
    }

    toDocument() {
        return {
            name: this.name,
            make: this.make,
            base_price: this.basePrice,
            final_price: this.finalPrice
        };
    }
}

module.exports = Product;
